### ISSUE TITLE
move source-map-loader into core

### ISSUE THREAD

**Issue by AvailCat** [webpack/webpack#10265]
# Bug report

**What is the current behavior?**
Bundle codes with eval sourcemap, but reference to a package that ships with standalone sourcemap will not convert standalone sourcemap to eval sourcemap, this will emit a lot of warning in browser console.

**If the current behavior is a bug, please provide the steps to reproduce.**
1. Import a package that ships with standalone sourcemap
2. Set `devtool: 'eval'` in webpack.config.js
3. Start with dev server
4. Check browser console

![image](https://user-images.githubusercontent.com/19658647/72504001-f0c61b00-3877-11ea-8b23-192ebc6cce55.png)

**What is the expected behavior?**
Standalone sourcemap in package should be converted to eval sourcemap or truncated

**Other relevant information:**
webpack version:  4.39.3 (Not able to tryout latest due typescript types issue)
Node.js version: 12.13.0
Operating System: Ubuntu 18.04.3
Additional tools:

**Comment by sokra** [comment]
Please try to use the `source-map-loader` to import the existing SourceMap.

**Comment by TrySound** [comment]
Hi @sokra. Is there a way to avoid installing one more package? I don't care about this sourcemaps. Everything worked fine before this release https://github.com/webpack/webpack/releases/tag/v4.41.5

**Comment by vkrol** [comment]
@TrySound It's not about webpack, it's about Chrome:
>  - We report more errors than previously, which is a WONTFIX.

https://bugs.chromium.org/p/chromium/issues/detail?id=1052872#c14

**Comment by evilebottnawi** [comment]
bump, I think we should improve it out of box

**Comment by TrySound** [comment]
I add this loader to solve the problem
```
module.exports = source => {
  return source.replace(/\n[^\n]*sourceMappingURL[^\n]*\n?/, '\n');
};
```

**Comment by evilebottnawi** [comment]
Yep, we need an option to import default behavior of existing source maps, possible situations:
- remove them if you don't need it (no warnings)
- handle them (move `source-map-loader` into core, logic it very simple, so it should be not hard)
- keep as is (maybe useful in some cases)

**Comment by sokra** [comment]
I think we can move this into the core.

**Comment by evilebottnawi** [comment]
Low priority, if anyone want to help, PR welcome

**Comment by TrySound** [comment]
Activate

**Comment by TrySound** [comment]
Oh no

**Comment by TrySound** [comment]
Still a problem
